### DISCUSSION TITLE
SRTP - RTP interworking

### DISCUSSION THREAD

**Discussion by cristeab** [hnimminh/libresbc#75] [Ideas]
Does this SBC support  SRTP - RTP conversions ? If not, is it possible to add this feature ?

**Comment by hnimminh** [comment]
@cristeab 
Yes, SRTP - RTP conversions is supported. Usually, SRTP go with TLS. 
I did not implement API to configure TLS certificate yet,  then it will require configure manually.

 You can file a feature request for API config TLS/SSL cert and document for TLS/SRTP configuration guide. I will work on that when I have free time.

**Comment by Jakihiko** [comment]
SRTP/TLS - RTP/UDP on my env work like charm
